Documentation/SpatialAwareness/SpatialObjectMeshObserver.md

```diff
@@ -0,0 +1,81 @@
+# The Spatial Object Mesh Observer
+
+The spatial object mesh observer is an editor-only data provider for the Spatial Awareness system 
+that enables using an imported 3D model to represent the spatial mesh. One common use of the 
+spatial object mesh observer is to use data scanned via a Microsoft HoloLens to test how an
+experience adapts to different environments from within Unity.
+
+## Getting started
+
+To enable the spatial object mesh observer, you will need to configure the spatial awareness system.
+
+1. Add a new spatial observer
+
+![Add Spatatial Observer](../../Documentation/Images/SpatialAwareness/AddObserver.png)
+
+> {!NOTE]
```

Comment:
Nit: I would delete this note (it's proliferated in other areas as well, but honestly I think right here it doesn't add that much.

Reply:
sounds good